package.json

```diff
@@ -62,9 +62,11 @@
     "lint:tests": "tslint -p tsconfig.json -t stylish -c src/tests/tslint.json -e src/tests/references.ts \"src/tests/**/*.ts\"",
     "prettier": "prettier --write \"**/*.{js,ts,tsx,json,md}\"",
     "start": "npm-run-all clean copy:json watch:src",
-    "test:mocha": "mocha \"dist/src/tests/**/*.js\" --timeout 5000",
+    "test:mocha": "mocha \"dist/src/tests/**/*.js\" --timeout 5000 --reporter mocha-junit-reporter",
```

Comment:
I think you'll want to take another look at the changes to these tasks. Seems like `test:mocha` changes should be reverted, as `test-junit` reports with the junit reporter?